easybuild/easyconfigs/i/imkl/imkl-2019.1.144-iompi-2019.01.eb

```diff
@@ -0,0 +1,39 @@
+# This is an easyconfig file for EasyBuild, see https://easybuilders.github.io/easybuild/
+
+name = 'imkl'
+version = '2019.1.144'
+
+homepage = 'http://software.intel.com/en-us/intel-mkl/'
+description = """Intel Math Kernel Library is a library of highly optimized,
+ extensively threaded math routines for science, engineering, and financial
+ applications that require maximum performance. Core math functions include
+ BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more."""
+
+toolchain = {'name': 'iompi', 'version': '2019.01'}
+
+source_urls = ['http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/13005/']
```

Comment:
@pescobar should be `http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/14895/` ?